README.md

```diff
@@ -28,15 +28,29 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. What is the DOM?
+**1. What is the DOM?**
 
-2. What is an event?
+The DOM stands for Document Object Model. It’s a language-neutral object representation of the HTML elements of a web page, and an API (which is an application program interface -- a set of routines, protocols, and tools, used to specify how software components interact successfully with each other). It's created by the browser when a web page is loaded. It has the structure of a tree, with nodes and objects. It allows access to the parts of the page and the ability to change them.
 
-3. What is an event listener?
+**2. What is an event?**
 
-4. Why would we convert a NodeList into an Array?
+An event is an action taken on an element of a web page, because of something a user or API does, or because something changes in an element. Some events are ‘click’, ‘mousedown’, ‘load’, and ‘touchstart.’ There's lots of them.
```

Comment:
An event can also be used to communicate anyway a _user interacts_ with a site. Your answer is absolutely correct. Just adding a bit to it in case it's asked in an interview.